.editorconfig

```diff
@@ -0,0 +1,6 @@
+# Some KtLint rules
+[*.{kt,kts}]
+indent_size=2
```

Comment:
Wondering if adding KtLint is also too opinionated for an MPP Template.

Setting `indent_size=2` is not even following the standard recommended Kotlin styleguide.

Reply:
Do you know the styleguide size?

Reply:
I personally don't see a big problem. And .editorConfig is for IDE's and not specific to ktlint

Reply:
Yes, but we also setup KtLint not only `.editconfig`.
By default the IntelliJ project wizard indeed adds `.editorconfig` with the following entry `kotlin.code.style=official`. 

I don't have a problem with `.editorconfig`, but adding KtLint brings similar problems as `buildSrc`. Auto Formatting is to this day still controversial, any many people don't like to use it.

Reply:
I see :D I removed ktlint